index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang= "en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portofolio</title>
+    <link rel="stylesheet" href="style.css">
+    </head>
+<body>
+<section class="Workssection1">
+    <header class="Header">
+        <a class="mylogo">MY logo</a>
```

Comment:
- [ Optional] Your Portfolio will look amazing if you replace this logo with your own logo.